command/hook_ui.go

```diff
@@ -316,10 +316,10 @@ func (h *UiHook) PostImportState(addr addrs.AbsResourceInstance, imported []prov
 	h.once.Do(h.init)
 
 	h.ui.Output(h.Colorize.Color(fmt.Sprintf(
-		"[reset][bold][green]%s: Import complete!", addr)))
+		"[reset][bold][green]%s: Import prepared!", addr)))
 	for _, s := range imported {
 		h.ui.Output(h.Colorize.Color(fmt.Sprintf(
-			"[reset][green]  Imported %s",
+			"[reset][green]  Prepared %s",
```

Comment:
```suggestion
			"[reset][green]  Prepared %s for import",
```

This is nitpicky, but what do you think?